modules/network-observability-enriched-flows.adoc

```diff
@@ -37,11 +38,29 @@ spec:
         targetHost: "ipfix-collector.ipfix.svc.cluster.local"
         targetPort: 4739
         transport: tcp or udp           <4>
-
-
+ -  type: OpenTelemetry                 <3>
+      openTelemetry:
+        targetHost: my-otelcol-collector-headless.otlp.svc
+        targetPort: 4317
+        type: grpc                      <5>
+        logs:                           <6>
+          enable: true
+        metrics:                        <7>
+          enable: true
+          prefix: netobserv
+          pushTimeInterval: 20s         <8>
+          expiryTime: 2m
+   #    fieldsMapping:                  <9>
+   #      input: SrcAddr
+   #      output: source.address
 ----
 <1> The Network Observability Operator exports all flows to the configured Kafka topic.
 <2> You can encrypt all communications to and from Kafka with SSL/TLS or mTLS. When enabled, the Kafka CA certificate must be available as a ConfigMap or a Secret, both in the namespace where the `flowlogs-pipeline` processor component is deployed (default: netobserv). It must be referenced with `spec.exporters.tls.caCert`. When using mTLS, client secrets must be available in these namespaces as well (they can be generated for instance using the AMQ Streams User Operator) and referenced with `spec.exporters.tls.userCert`.
-<3> You can export flows to IPFIX instead of or in conjunction with exporting flows to Kafka.
+<3> You can export flows to IPFIX, OpenTelemetry, and Kafka individually or concurrently. 
 <4> You have the option to specify transport. The default value is `tcp` but you can also specify `udp`.
-. After configuration, network flows data can be sent to an available output in a JSON format. For more information, see _Network flows format reference_.
+<5> The protocol of OpenTelemetry connection. The available options are `http` and `grpc`.
+<6> OpenTelemetry configuration for exporting logs, which are the same as the logs created by Loki. 
+<7> OpenTelemetry configuration for exporting logs, which are the same as the metrics created by Prometheus, which are specified in the `spec.processor.metrics.includeList` of the `FlowCollector` and any custom metrics you defined using the `FlowMetrics` custom resource.
+<8> The time interval that metrics are sent to the OpenTelemetry collector.
+<9> Network Observability network flows formats get automatically renamed to an OpenTelemetry compliant format. The `fieldsMapping` specification gives you the ability to customize the OpenTelemetry format output. For example in the YAML sample, `SrcAddr` is the Network Observability input field, and it is being renamed `source.address` in OpenTelemetry output. You can see both Network Observability and OpenTelemetry formats in the "Network flows format reference".
+. After configuration, network flows data can be sent to an available output in a JSON format. For more information, see "Network flows format reference".
```

Comment:
This last step doesn't really instruct users to do anything, I think this can just be a regular sentence at the end of the procedure:
```suggestion

After configuration, network flows data can be sent to an available output in a JSON format. For more information, see "Network flows format reference".
```

Reply:
Agree, and removed this from the steps.